packages/manager/modules/netapp/src/dashboard/volumes/dashboard/snapshots/Snapshot.class.js

```diff
@@ -23,4 +23,8 @@ export default class Snapshot {
   isPending() {
     return STATUS.PENDING.includes(this.status);
   }
+
+  isBeingDeleted() {
```

Comment:
may be we can rename it `isDeleting` to be compliant with otherers function (eg: `isPending`) ?